library/init/meta/interactive.lean

```diff
@@ -133,813 +132,3 @@ meta def param_desc : expr → tactic format
   then return $ to_fmt "{ tactic }"
   else paren <$> pp e
 end interactive
-
```

Comment:
I am fine with reducing dependencies, but a new directory `tactic` with a single `interactive.lean` file looks strange. I think `interactive_base.lean` and `interactive.lean` in the same directory (i.e., `meta`) looks better. 

BTW, I'm not against mirroring the namespace hierarchy in the directory structure. However, if you do it, we should be consistent. We should also do it in a different PR. What do you think?